src/Features/Core/Portable/ChangeSignature/ChangeSignatureCodeAction.cs

```diff
@@ -30,7 +30,11 @@ internal class ChangeSignatureCodeAction(AbstractChangeSignatureService changeSi
         public override object? GetOptions(CancellationToken cancellationToken)
             => AbstractChangeSignatureService.GetChangeSignatureOptions(_context);
 
-        protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken)
+        protected override Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken)
+            => throw ExceptionUtilities.Unreachable();
+
+        private protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(
+            object options, IProgress<CodeAnalysisProgress> progressTracker, CancellationToken cancellationToken)
```

Comment:
CodeActionWithOptions was updated to take IProgress as well (just like CodeAction).